eng/scripts/Language-Settings.ps1

```diff
@@ -4,31 +4,27 @@ $PackageRepository = "NPM"
 $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
+$PackageRegex = "/\@(.*)\//i"
+$PackageReplacement = "`$1-"
 
-function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
-{
+function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName) {
```

Comment:
For multi-line blocks like functions I suggest keeping the '{' on its own line so it easier to track the block.

Reply:
I have code format of the js repo folder. I will turn it off.